<ui version="4.0" >
 <class>AddressBookSelectorWidget_Base</class>
 <widget class="QWidget" name="AddressBookSelectorWidget_Base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>572</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Select Contact</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="3" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>405</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <widget class="QPushButton" name="addAddresseeButton" >
     <property name="toolTip" >
      <string>Create a new entry in your address book</string>
     </property>
     <property name="text" >
      <string>Create New Entr&amp;y...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" native="1" name="lblHeader" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="K3ListView" name="addresseeListView" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Select the contact you want to communicate with via Instant Messaging</string>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <property name="resizeMode" >
      <enum>Q3ListView::LastColumn</enum>
     </property>
     <column>
      <property name="text" >
       <string>Photo</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Email</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lblSearch" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>S&amp;earch:</string>
       </property>
       <property name="buddy" >
        <cstring>kListViewSearchLine</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="K3ListViewSearchLine" native="1" name="kListViewSearchLine" />
     </item>
    </layout>
   </item>
  </layout>
 </widget>
  <customwidgets>
  <customwidget>
   <class>K3ListView</class>
   <extends>Q3ListView</extends>
   <header>k3listview.h</header>
  </customwidget>
  <customwidget>
   <class>K3ListViewSearchLine</class>
   <extends></extends>
   <header>k3listviewsearchline.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>addresseeListView</tabstop>
  <tabstop>addAddresseeButton</tabstop>
 </tabstops>
 <includes>
  <include location="global" >k3listviewsearchline.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
